### PR TITLE
bugfix for not-Geoaxes

### LINKED ISSUE #106: Plotting error due to no cartopy projection
The Iris sample data files atlantic_profiles and hybrid_height produce the following error when you try and plot them in the Cube Explorer:

None-type object has no attribute 'as_cartopy_projection'

### PR DESCRIPTION
**Pull request by corinnebosley**
Closes #106

### CODE REVIEW

**Comment by corinnebosley** [comment]
Closes #110

**Comment by corinnebosley** [comment]
Closes #112 (maybe)

**Comment by corinnebosley** [comment]
@dkillick Thanks for the pointer!

File: lib/cube_browser/explorer.py [hunk 1]
```diff
@@ -258,9 +260,8 @@ def _goplot(self, sender):
                 x_name = cube.coord(axis='X').name()
                 y_name = cube.coord(axis='Y').name()
                 if x_name == pc_x_name and y_name == pc_y_name:
-                    projection = cube.coord_system().as_cartopy_projection()
-                    ax = fig.add_subplot(sub_plots + spl,
-                                         projection=projection)
+                    proj = iplt.default_projection(cube) or ccrs.PlateCarree()
```
Comment (dkillick) [inline lib/cube_browser/explorer.py]: What if you wanted to specify your own projection for the data? Or is that not yet an option for Explorer?

Comment (corinnebosley) [inline lib/cube_browser/explorer.py]: I can see Mark saying "keep it clean", so I would say that is not an option at this point.

**Comment by dkillick** [comment]
Nice - a good small change @corinnebosley :clap:

**Pull request merged**